sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Internal/ValidationOptions.cs

```diff
@@ -60,13 +61,17 @@ public void Add(string connectionString)
             _hostKeyMappings.Add(host.Host, accessKey);
         }
 
-        internal bool ContainsHost()
+        public void Add(Uri endpoint)
         {
-            return _hostKeyMappings.Count > 0;
+            _hostKeyMappings.Add(endpoint.Host, null);
         }
 
-        internal bool ContainsHost(string host)
+        internal bool ContainsHost(string host = null)
         {
+            if (host == null)
```

Comment:
in which case is host `null`?

Reply:
Here merged the methods, `null` means default `ValidationOptions` is created without endpoint information added. See [Usage](https://github.com/Azure/azure-sdk-for-net/blob/1352d0a8ac49a3425fee62061e3fc48b13e8f141/sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Internal/WebPubSubMiddleware.cs#L46)